chroma_core/models/host.py

```diff
@@ -111,13 +111,6 @@ class ManagedHost(DeletableStatefulObject):
         default=False, help_text="True if there are package updates available for this server"
     )
 
-    client_filesystems = models.ManyToManyField(
-        "ManagedFilesystem",
-        related_name="workers",
-        through="LustreClientMount",
-        help_text="Filesystems for which this node is a non-server worker",
-    )
-
```

Comment:
This was a dead piece of code

Reply:
This is missing a corresponding migration.

Reply:
`./manage.py makemigrations` does not notice it, and there is not `client_filesystem` fields in the `chroma_core_managedhost` table.

Reply:
I overlooked it : `migrations.RemoveField(model_name="managedhost", name="client_filesystems",),` :)
It may be argued to be in a separate migration file or even PR, but this this is a consequence of the main gain (removal of the foreign key), it might rest in here as well.

Reply:
I'm ambivalent if it needs to be in a separate file, but a field change and cooresponding migration must be in the same patch. Otherwise everyone using the patch will have out of date migrations.

Reply:
Yes, it is in the migration file attached to this PR.

Reply:
Ah, sorry missed that.